config/locales/de.yml

```diff
@@ -0,0 +1,204 @@
+        title: Zurücksetzen des Passworts
+        text: "Brauchst du ein neues Passwort? Kein Problem. %{link}"
+        link_text: Jetzt ein neues Passwort zuschicken lassen.
+      api_access:
+        title: API-Zugang
+        key_is: "Dein API-Schlüssel ist %{key}."
+        credentials: "Wenn du %{gem_commands_link} über das Terminal oder über das CMD verwenden möchtest, dann brauchst du die %{gem_credentials_file}-Datei, die du mit dem folgenden Befehl erzeugen kannst:"
+        link_text: Gem-Befehle
+        reset: Zurücksetzen meines API-Schlüssels
+        confirm_reset: Bist du sicher? Das kann nicht rückgängig gemacht werden.
+
+  rubygems:
+    dependencies:
+      header: "%{title} Abhängigkeiten"
+    edit:
+      title: "Bearbeite Links"
+      subtitle: "für %{gem}"
+      other_fields_notice: Alle anderen Felder können nur durch eine gepushte, aktualisierte Version deines Gems bearbeitet werden.
+    index:
+      title: 'Gems'
+      downloads: Downloads
+    show:
+      not_hosted_notice: Dieses Gem wird nicht aktuell auf RubyGems.org gehostet.
+      install: installieren
+#      yanked_notice: "This gem has been yanked, and it is not available for download directly or for other gems that may have depended on it."
+      authors_header: Autoren
+      downloads_for_this_version: "Für diese Version"
+      owners_header: Besitzern
+      links:
+        header: Links
+        home: Startseite
+        code: Quellcode
+        docs: Dokumentation
+        wiki: Wiki
+        mail: Mailingliste
+        bugs: Bug Tracker
+        download: Download
+        subscribe: Abonniere
+        unsubscribe: Storniere
+        report_abuse: Missbrauch melden
+        badge: Abzeichen
+        rss: RSS
+      versions_header: Versionen
+      bundler_header: Gemfile
+      show_all_versions: "Zeige alle Versionen (%{count} total)"
+      licenses_header: Lizenz
+      no_licenses: k.A.
+      requirements_header: Anforderungen
+      sha_256_checksum: SHA 256-Prüfsumme
+      copy_to_clipboard: In die Zwischenablage kopieren
+      copied: Kopiert!
+      required_ruby_version: Erforderliche Ruby-Version
+
+  searches:
+    show:
+      subtitle: "für %{query}"
+      exact_match: Exakte Übereinstimmung
+
+  sessions:
+    new:
+      forgot_password: "Passwort vergessen?"
+
+  stats:
+    index:
+      total_downloads: Downloads insgesamt
+      total_gems: Gems insgesamt
+      total_users: Gems insgesamt
+      all_time_most_downloaded: Am meisten Heruntergeladen in der gesamten Zeit
+
+  users:
+    new:
+      have_account: "Hast du schon ein Account?"
+
+  versions:
+    index:
+      title: "Alle Versionen von %{name}"
+      versions_since: "%{count} Versionen seit %{since}"
+      not_hosted_notice: Dieses Gem wird nicht gerade von RubyGems.org gehostet.
+    version:
+#      yanked: "yanked"
+
+  activerecord:
+    attributes:
+      user:
+        avatar: Avatar
+        email: E-Mail-Adresse
+        handle: Verhandele
```

Comment:
yeah no :) this is the german word for "to handle", but not for the noun "handle". This should be something like "Kürzel" or "Benutzername" (username).